### DISCUSSION TITLE
Get notifications option for poll participants

### DISCUSSION THREAD

**Discussion by ionarn** [lukevella/rallly#444] [Ideas]
Yesterday someone told me, that getting notifications when something changes in the poll could be useful to stay updated.
Maybe if participants enter an e-Mail, they then could get updated with e-Mails. I read in the comments of another idea that sending e-Mails is rather expensive. So maybe this could be something for self-hosted or paid version of rallly.

Another way could be to have an app that lists all the polls one is participating in and gives notifications via Android/iOS when something changes or happens in the poll.
But just for notifications an app is most likely too much work.

What do you think?

**Comment by lukevella** [comment]
Hi there. Yes, totally agree. The next milestone is focused around allowing administrators to finalize a poll by selecting a date and part of this involves notifying participants as well. I've moved to a different email provider which has reduced the cost of sending out emails so hopefully this can be offered for free as well.
